### PR TITLE
Timeline

### PR DESCRIPTION
**Pull request by skondkar**

### CODE REVIEW

**Comment by hawkularqe** [comment]
Test FAILed.<br>
BuildID: [220](http://jenkins.jonqe.lab.eng.bos.redhat.com:9080/job/miq-mw-ui-tests/220/)
<br> Gif records:<br> - [record_test_datasource.flv](http://10.16.23.184/records/220/record_test_datasource.flv) <br> - [record_test_deployments.flv](http://10.16.23.184/records/220/record_test_deployments.flv) <br> - [record_test_domains.flv](http://10.16.23.184/records/220/record_test_domains.flv) <br> - [record_test_download_report.flv](http://10.16.23.184/records/220/record_test_download_report.flv) <br> - [record_test_providers.flv](http://10.16.23.184/records/220/record_test_providers.flv) <br> - [record_test_servers.flv](http://10.16.23.184/records/220/record_test_servers.flv) <br> - [record_test_timeline.flv](http://10.16.23.184/records/220/record_test_timeline.flv) <br> - [record_test_topology.flv](http://10.16.23.184/records/220/record_test_topology.flv) <br>

**Comment by mattmahoneyrh** [comment]
Noticed that setup.sh was not the latest version (should be "pip install mgmtsystem==1.3.10").
Please do the following, if not already done, to validate that the script works as expected:

 - rm -rf .cf-ui
 - setup.sh
 - run test_timeline.py    (with the latest mgmtsystem)

File: views/timelines.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        self.web_driver.find_element_by_xpath("//a[contains(@id,'timeline')]").click()
+        ui_utils(self.web_session).sleep(15)
+
+    def verify_event(self, event_type):
+
+        # Verify event where type for successful event is 'ok' and for unsuccessful event is 'error'
+
+        eap = servers(self.web_session).find_non_container_eap_in_state("running")
+        eap_name = eap.get('Server Name')
+        print eap_name
```
Comment (mattmahoneyrh) [inline views/timelines.py]: Either delete the print - or change to using logger

File: views/timelines.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        ui_utils(self.web_session).sleep(15)
+        return True
+
+    def navigate_to_timeline(self):
+
+        self.web_session.web_driver.get("{}//ems_middleware/show_list?type=list".format(self.web_session.MIQ_URL))
+        ui_utils(self.web_session).click_on_row_containing_text(self.web_session.HAWKULAR_PROVIDER_NAME)
+        self.web_driver.find_element_by_xpath("//button[@title='Monitoring']").click()
+        self.web_driver.find_element_by_xpath("//a[contains(@id,'timeline')]").click()
+        ui_utils(self.web_session).sleep(15)
```
Comment (mattmahoneyrh) [inline views/timelines.py]: Is there either an element or text on page that can be used for a wair-for, instead of flat 15 seconds?

**Pull request merged**